doc/services/device_mgmt/ec_host_cmd.rst

```diff
@@ -47,8 +47,9 @@ The supported backend and peripheral drivers:
 Initialization
 **************
 
-If the application configures one of the following backend chosen nodes, then the corresponding backend
-initializes the host command subsystem by calling :c:func:`ec_host_cmd_init`:
+If the application configures one of the following backend chosen nodes and
+:kconfig:option:`CONFIG_EC_HOST_CMD_INITIALIZE_AT_BOOT` is set, then the corresponding backend initializes the host
```

Comment:
Nit - line length should be less than 100.

Reply:
Done.